test/windowclose.jl

```diff
@@ -11,6 +11,9 @@ window = GLFW.CreateWindow(800, 600, "InexactError")
 ccall( (:glfwSetWindowShouldClose, GLFW.lib), Void, (GLFW.WindowHandle, Cint), window, 189)
 @test GLFW.WindowShouldClose(window)
 
+sleep(0.25)
```

Comment:
Please remove. If it doesn't work without a sleep, that could be indicative of other problems.